core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.spec.ts

```diff
@@ -151,6 +178,14 @@ describe('DotDeviceSelectorSeoComponent', () => {
         expect(link.properties.href).toContain('/c/content');
     });
 
+    it('should not have a link to add device', async () => {
+        spyOn(dotCurrentUserService, 'getCurrentUser').and.returnValue(of(CurrentUserDataMock));
+        component.ngOnInit();
```

Comment:
can we use a detect changes instead of this? I think is not recommended to call angular methods like this